### ISSUE TITLE
Unable to start

### ISSUE THREAD

**Issue by Michas0206** [retrocomputacion/retroterm#8]
Hi,
I am unable to start Retroterm at all, it just 'freezes' / hangs here:
![IMG_20241020_025703_DRO](https://github.com/user-attachments/assets/28d5a3df-13b9-413a-ae99-c09772577825)
I am able to exit with Runstop or enter setup with C= +F7, but thats it...I cannot enter any commands, nothing happens anymore.

![IMG_20241020_025720_DRO](https://github.com/user-attachments/assets/80242aef-161b-49db-993f-cd2ab9b2b312)
![IMG_20241020_025735_DRO](https://github.com/user-attachments/assets/b2fc3b2e-ec81-4bc7-b0d5-11de0ec4e37c)

Any ideas what I am doing wrong?

By the way, a different Terminal program (Ultimate Term) works fine with exactly these settings, I can connect up to any BBS.

Trying the SL-version of Retroterm makes no difference, it hangs at the same point and nothing happens anymore.

**Comment by retrocomputacion** [comment]
I don't have a network enabled Ultimate to try myself, but your settings look correct to what I know works for other users (other than 'Set socket opt...', but I don't think that would be the problem here)
Did you try sending a command even if nothing prints on screen when you type?
For example ATI<return>

**Comment by Michas0206** [comment]
I assume this can be closed, I assume an issue in Ultimate firmware 3.10c.
I am using the old U2 (without plus) and for a long time, 3.10c was the latest and best firmware.

After I experienced this tonight I gave Gideons newer 3.11-firmware a try - and now it does not freeze anymore, on fw 3.11 I could enter commands at least.

I have had no known commands in my head/never used this before and have to figure them out now, I turned off the c64 because it was very late tonight and going to do further tests today but as said, it looks better with fw 3.11 already.
